code/reports/PagesDueForReviewReport.php

```diff
@@ -153,6 +155,11 @@ public function columns()
             ));
         }
 
+        // Only show pages assigned to the current user?
+        if (!empty($params["OnlyMyPages"])) {
+            $records = $records->filter('ContentReviewUsers.ID', Member::currentUserID());
```

Comment:
This won't work if I'm in a group that's a reviewer of this page only. It needs to handle both I'm afraid.

You can solve this by first evaluating all group IDs that the current user is in, then using filterAny() on both ContentReviewUsers.ID and ContentReviewGroups.ID